src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/EligibilityService.kt

```diff
@@ -99,4 +100,8 @@ class EligibilityService(
     // Trust the Nomis recall flag as a fallback position - the above rules should always override
     return@early it.recall ?: error("${it.prisonerNumber} missing recall flag")
   }
+
+  private fun isBreachOfTopUpSupervision(): EligibilityCheck = early@{
+    it.imprisonmentStatus != null && it.imprisonmentStatus == "BOTUS"
```

Comment:
don't need the null check as `it.imprisonmentStatus == "BOTUS"` will equal false when `imprisonmentStatus` is null